lib/fetch/body.js

```diff
@@ -234,8 +234,8 @@ function extractBody (object, keepalive = false) {
         }
         return controller.desiredSize > 0
       },
-      async cancel (reason) {
-        await iterator.return()
+      cancel (reason) {
+        return iterator.return()
```

Comment:
This might hide it from the stacktrace, not sure the attempt is to keep that function hidden as is part of the `RedeableStream` API and can be helpful while debugging.